blog/models.py

```diff
@@ -30,3 +30,26 @@ def publish(self):
     def __str__(self):
         """Show post title."""
         return self.title
+
+    def approved_comments(self):
+        """Return all approved(published) comments."""
+        return self.comments.filter(approved_comment=True)
+
+
+class Comment(models.Model):
+    """Represents a comment to post."""
+
+    post = models.ForeignKey('blog.Post', related_name='comments')
+    author = models.CharField(max_length=200)
+    text = models.TextField()
+    created_date = models.DateTimeField(default=timezone.now)
+    approved_comment = models.BooleanField(default=False)
```

Comment:
It's logical to call this column `is_approved`. There's no need to repeat `comment`keyword :)